File: CodeWars/.js/8kyu-Reversed-Sequence.js
// Build a function that returns an array of integers from n to 1 where n>0.

// Example : n=5 --> [5,4,3,2,1]

const chai = require("chai");
const assert = chai.assert;
chai.config.truncateThreshold=0;

describe("reverseSeq", function() {
  it("Sample Test", function() {
    assert.deepEqual(reverseSeq(5), [5, 4, 3, 2, 1]);
  });
});

// my code

const reverseSeq = n => {
    let arr=[]
    for (let i=0;i<n;i++){
      arr.push(n-i)
    }
    return arr
  };

File: CodeWars/.cs/8kyu-ReversedSequence.cs
// Build a function that returns an array of integers from n to 1 where n>0.

// Example : n=5 --> [5,4,3,2,1]

namespace Solution 
{
  using NUnit.Framework;
  using System;

  [TestFixture]
  public class SolutionTest
  {
    [Test]
    public void SampleTest()
    {
      Assert.That(Kata.ReverseSeq(5), Is.EqualTo(new int[] {5,4,3,2,1}));
    }
  }
}

// my code

using System.Collections.Generic;

public static class Kata
{
  public static int[] ReverseSeq(int n)
  {
    int[] returnArray = new int[n];
    for (int i = 0; i < n; i++)
    {
      returnArray[i] = n-i;
    }
    return returnArray;
  }
}